src/redux/filter/filter.types.js

```diff
@@ -0,0 +1,15 @@
+export const SET_CURRENT_PAGE = 'SET_CURRENT_PAGE';
```

Comment:
Please, reduce the number of actions to the one `setFilter` that takes an object with specific keys and merge that action with products. Actions, states, types, and saga should be part of products. It is the same thing